tform/aws-resources/main.tf

```diff
@@ -0,0 +1,70 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.27"
+    }
+    github = {
+      source  = "integrations/github"
+      version = "~> 4.0"
+    }
+  }
+
+  required_version = ">= 0.14.9"
```

Comment:
It is advice to use a pinned version for reproduce-ability

Reply:
Will do

Reply:
pinned